src/Components/SmartComponents/Table/BaseTableToolbar.js

```diff
@@ -18,7 +18,11 @@ const BaseTableToolbar = ({
     <PrimaryToolbar
       pagination={
         isLoading ? (
-          <Skeleton fontSize="xl" width="200px" style={{ margin: 10 }} />
+          <Skeleton
+            fontSize="xl"
+            width="200px"
+            style={{ marginTop: 8, marginRight: 16 }}
```

Comment:
Why do we need this specific margins? would not it result in discrepancy with other apps? if not, can we use pf variables for this purpose?

Reply:
You are right, I removed these margins and now it's consistent with Inventory and other apps